test/typescript.spec.js

```diff
@@ -2,7 +2,7 @@ import * as tt from 'typescript-definition-tester'
 
 
 describe('TypeScript definitions', function () {
-  this.timeout(0)
+  //this.timeout(0)
```

Comment:
According to the [Mocha homepage](https://mochajs.org/):
> Use `this.timeout(0)` to disable timeouts all together

On Travis (and my machine) the new Jest test passes without hitting the default timeout. Thus, in my opinion, the line should be removed.

But if I'm missing anything, let me know.